docs/rst/tutorials/core/qos/persistency/persistency.rst

```diff
@@ -125,27 +129,31 @@ when a publisher crashes.
 
     .. code-block:: bash
 
-        1$ export FASTRTPS_DEFAULT_PROFILES_FILE=/home/tutorial/writer_config.xml
-        1$ ros2 topic pub --times 5 --qos-durability system_default /persistency_test std_msgs/String "{data: 'Hello'}"
+        # Terminal 1
+        export FASTRTPS_DEFAULT_PROFILES_FILE=/home/tutorial/writer_config.xml
+        ros2 topic pub --times 5 --qos-durability system_default /persistency_test std_msgs/String "{data: 'Hello'}"
```

Comment:
Consider including ```docker run -ti --name persistence_testing ubuntu-vulcanexus:humble-micro``` command first. Something "from sratch" may be useful for users entering directly to this this section. 

If it is accepted, we could replicate it to the subscriber example.

Reply:
I don't see it. Besides, this is the same structure we used in the [Fast-DDS examples](https://fast-dds.docs.eprosima.com/en/latest/docker/fastdds/fast_dds.html).